replicator/src/main.rs

```diff
@@ -68,6 +77,12 @@ fn main() {
     } else {
         Keypair::new()
     };
+    let storage_mining_pool_pubkey =
+        matches
+            .value_of("storage_mining_pool_pubkey")
+            .map(|pubkey_str| {
```

Comment:
Duplicate this helper function into this file for better error handling: https://github.com/solana-labs/solana/blob/master/install/src/lib.rs#L28

Reply:
thank you for this. much better